backend/user_ops.py

```
@@ -0,0 +1,129 @@
+        user_id = await database.execute(
+            """INSERT INTO users (github_id, login, name, email, avatar_url, access_token)
+               VALUES (%s, %s, %s, %s, %s, %s)""",
+            (github_id, login, name, email, avatar_url, access_token)
+        )
+        
+        user = await database.fetchone(
+            "SELECT * FROM users WHERE id = %s", (user_id,)
+        )
+        
+        logger.info(f"Created new user: {login} (id={user_id})")
+        return {"user": user, "created": True}
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)